docs/api.apib

```diff
@@ -55,12 +79,13 @@ Creates a new image based on a JSON object containing the following metadata.
             Authorization: Bearer JWT
     
     + Body
-        {
-            "label": "FW_Example",
-            "fw_version": "1.0.0",
-            "hw_version": "1.0.0",
-            "sha1": "cf23df2207d99a74fbe169e3eba035e633b65d94"
-        }
+    
+            {
+                "label": "FW_Example",
+                "fw_version": "1.0.0",
+                "hw_version": "1.0.0",
```

Comment:
I strongly agree that device manager should maintain the hw_version info, but I disagree about the template comment.
Some times, you will want to have multiple revisions of the same hardware (especially during development) where each revision may just replace a resistor (to increase a given current or tweak a sensor etc) or change the manufacturer of some component. The result will be the exact same software with the exception of a hardware abstraction layer or even just one or two constants. In this case it is very useful to have multiple image with the same fw_version but different hw_versions

Reply:
Let me see if I'm getting this straight:
>The result will be the exact same software with the exception of a hardware abstraction layer or even just one or two constants.

So we're talking about two different instances of images - each with different sw, hw versions, different binaries (the image has to be updated to include this changed HAL or constants) under the same label (probably?)

Reply:
No, we are talking about two different instances with the same label and sw version but a different hw version, assuming we are using semantic versioning. 
I mean, surely we could encode all this information in the sw_version like 1.0.0-hw-A and 1.0.0-hw-B and drop the hw_version value, but it is not a good idea to encode it in the version itself like 1.0.1 and 1.0.2 because they don't necessarily are incremental changes.

Reply:
If we're following [semver](https://semver.org/#spec-item-10), wouldn't that be mapped as `M.m.b+hardwareinfo` ?

Reply:
I will agglutinate both fields in another pull request

Reply:
ok!